src/Nethermind/Ethereum.Blockchain.Block.Legacy.Test/BlockGasLimitTests.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Ethereum.Test.Base;
+using NUnit.Framework;
+
+namespace Ethereum.Blockchain.Block.Legacy.Test
+{
+    [TestFixture]
+    [Parallelizable(ParallelScope.All)]
+    public class BlockGasLimitTest : LegacyBlockchainTestBase
+    {
+        [TestCaseSource(nameof(LoadTests))]
+        public async Task Test(LegacyBlockchainTest test)
+        {
+            await RunTest(test);
+        }
+
+        public static IEnumerable<LegacyBlockchainTest> LoadTests() { return new DirectoryTestsSource("bcBlockGasLimitTest").LoadLegacyTests(); }
```

Comment:
Those classes are basically the same with only different directory? Can we remove this duplication?

Reply:
this the main reason why PR is still under WIP,
my next goal is to make the older ones to get up to date tests

Reply:
just wanted to show progress tbh and let Tomek run those tests in Blockchain.Legacy.Test

Reply:
ok, use Draft PR in future.